### DISCUSSION TITLE
[V5] Oracle - Sequence

### DISCUSSION THREAD

**Discussion by gedsonmarcelino** [adonisjs/core#1779] [Help]
Hi guys,

I am trying to insert a record in oracle database, but I dont know how to insert a sequence in this new version (Adonis 5).

Following the error and code:

```
// Model
@column({ isPrimary: true })
public ID: any;

// Code
await MyModel.create({ ID: "TABLE.NEXTVAL", ... });

// Error
insert into "TABLE" ("ID", ... ) values (:1, ...) returning "ID" into :25 - NJS-044: bind object must contain one of the following keys: "dir", "type", "maxSize", or "val"
```
could someone help me?

I've tried to use raw, but without success as well:
```
// Code
await MyModel.create({ ID: Database.raw("TABLE.NEXTVAL"), ... });
```

**Comment by thetutlage** [comment]
The `TABLE.NEXTVAL` is treat as a string and not the Oracle expression. Is there any reason that this is not defined at the table level?
